lib/streamlit/__init__.py

```diff
@@ -101,6 +101,14 @@ def _update_logger() -> None:
     _logger.init_tornado_logs()
 
 
+# TODO(willhuang1997): Make sure this docs link gets updated for get and set query params.
+_experimental_query_params_deprecate_msg = (
+    "Please refer to our [docs page](%SOME_DOC_LINK%) to replace"
+)
+_EXPERIMENTAL_GET_QUERY_PARAMS = "experimental_get_query_params"
+_EXPERIMENTAL_SET_QUERY_PARAMS = "experimental_set_query_params"
```

Comment:
These intermediary variables just make things harder to read because they add unnecessary indirection and are defined far away from where they're actually used / they're not really going to get much reuse if any. There's not really any reason that we can't just use these string literals directly in the calls to `_deprecate_func_name`, and it'd be cleaner to do that.

Also, why are some variables `UPPER_CAMEL_CASE` and others `lower_camel_case`?